src/app/modules/form/admin-form/admin-form.service.ts

```diff
@@ -1188,9 +1188,13 @@ export const createTwilioCredentials = (
   const formId = form._id
 
   const msgSrvcName = `formsg/${config.secretEnv}/form/${formId}/twilio`
+
   const body: CreateSecretRequest = {
     Name: msgSrvcName,
     SecretString: twilioCredentialsData.toString(),
+    Description: `autogenerated via API on ${Math.floor(
+      Date.now() / 1000,
```

Comment:
should we just use Date.now() directly instead of rounding? Better yet, can we use the ISO string? `Date.toISOString()`